src/main.c

```diff
@@ -137,6 +142,11 @@ static bool parse_args(int argc, char **args)
         case 'm':
             opt_misaligned = true;
             break;
+        // case 'p':
```

Comment:
It does not work.
```
$ build/rv32emu -p build/dhrystone.elf 
RV32I[MA] Emulator which loads an ELF file to execute.
Usage: ./build/rv32emu [options] [filename] [arguments]
Options:
  -t : print executable trace
  -d [filename]: dump registers as JSON to the given file or `-` (STDOUT)
  -q : Suppress outputs other than `dump-registers`
  -a [filename] : dump signature to the given file, required by arch-test test
  -m : enable misaligned memory access
  -p : print profiling data
  -h : show this message
```

Reply:
I have uncommented, the profiling data log works for interpreter mode and JIT mode now. Please check the profiling data log meets the requirement ot not.

* command
```
./build/rv32emu -p prof_log ./build/pi.elf
```

Reply:
> `build/rv32emu -p prof_log ./build/pi.elf`

There is no need to specify a distinct file name for the profiling data. You can just add a suffix, like `.prof`, to the existing file name. At the same time, make sure to create a header for the profiling data, enabling it to be easily identified by the profiler.